CodePush.m

```diff
@@ -36,26 +36,41 @@ @implementation CodePush {
 // that is associated with an update's package.
 static NSString *const PackageHashKey = @"packageHash";
 static NSString *const PackageIsPendingKey = @"isPending";
+static NSString *const BinaryBundleDateKey = @"binaryDate";
+
+// These values are used to save the bundleURL and extension for the JS bundle
+// in the binary.
+static NSString *binaryJsName = @"main";
+static NSString *binaryJsExtension = @"jsbundle";
+
 
 #pragma mark - Public Obj-C API
 
++ (NSURL *)binaryJsBundleUrl
```

Comment:
Maybe call this `binaryBundleURL`? We should definitely at least capitalize the `URL` part since that is the Cocoa Touch convention.